src/script/cpp_api/s_server.cpp

```diff
@@ -168,25 +171,26 @@ void ScriptApiServer::on_shutdown()
 	runCallbacks(0, RUN_CALLBACKS_MODE_FIRST);
 }
 
-std::string ScriptApiServer::formatChatMessage(const std::string &name,
-	const std::string &message)
+std::wstring ScriptApiServer::formatChatMessage(const std::wstring &name,
```

Comment:
this could be moved out of `ScriptApiServer` now that it no longer calls Lua

Reply:
Where to?